frontend/src/bundles/common/components/header/header.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box, Flex, Text } from '@chakra-ui/react';
+
+type Properties = {
+    left?: React.ReactNode;
+    center?: React.ReactNode;
+    right?: React.ReactNode;
+};
```

Comment:
if for example you will pass left and center only it will display center more towards the right side not directly in center because of justify-content: space-between
Am I right?

Reply:
I had idea to make it as grid, but I am not sure how should I split space between these columns (equally or should I give more space to the center component).